TicTacToe.sol

```diff
@@ -40,37 +13,39 @@ contract TicTacToe {
         int8[9] board;
         address winner;
         bool isGameFinished;
+        uint8 movesMade;
```

Comment:
By tracking the number of moves made and incrementing in `makeMove`, we can remove the entire `isBoardFull` function and save gas by not having to `for` loop over the entire board